### ISSUE TITLE
Tribble type error with gatk-framework and CombineVariants

### ISSUE THREAD

**Issue by lbeltrame** [chapmanb/bcbio-nextgen#765]
This one is really cryptic... 

````
MESSAGE: Invalid command line: No tribble type was provided on the command line and the type of the file could not be determined dynamically.
````

This occurs after VEP annotation, with latest (I think) master. Example command line

````
/mnt/data/bcbio-tools/bin/gatk-framework -Xms750m -Xmx29484m -XX:+UseSerialGC -XX:+UseSerialGC -XX:+UseSerialGC -U LENIENT_VCF_PROCESSING --read_filter BadCigar --read_filter NotPrimaryAlignment -T CombineVariants -R /mnt/data/bcbio/genomes/Hsapiens/hg19/seq/hg19.fa --out /mnt/data/projects/OV202/BACO/work/gemini/tx/tmpz2k4iS/BACO_tumor1_vs_normal-mutect-nomultiallelic.vcf.gz --variant:v0 /mnt/data/projects/OV202/BACO/work/gemini/BACO_tumor1_vs_blood-mutect-biallelic.vcf.gz --variant:v1 /mnt/data/projects/OV202/BACO/work/gemini/BACO_tumor1_vs_normal-mutect-multiallelic-decompose-vepeffects.vcf.gz --rod_priority_list v0,v1 --suppressCommandLineHeader --setKey null -L /mnt/data/projects/OV202/BACO/work/bedprep/OV202.bed --interval_set_rule INTERSECTION -nt 4
````

However the VCF files look OK to me....

**Comment by lbeltrame** [comment]
Just in case, I've wiped the work directory and restarted the run from scratch. Let's see if it fails again...

**Comment by lbeltrame** [comment]
I noticed it only occurs with "biallelic" variants. However, since I've been out of the loop for a while, I have no idea how these files are generated.

**Comment by chapmanb** [comment]
Luca;
Sorry about the problem and thanks for the heads up. This is new code that handles decomposing multiallelic variants so they'll be correctly loaded into GEMINI. It looks like VEP produces a completely empty output file if the input file has no variants. As you suggested, I added a check for this and it should now run cleanly. Please let me know if this fixes #766 as well or we need to dig more into that problem. Thanks again.

**Comment by lbeltrame** [comment]
> This is new code that handles decomposing multiallelic variants so they'll be correctly loaded into GEMINI

Perhaps OT, but thanks, like this one more bit of custom code I'm using bites the dust. ;)
I'll check in a bit: I work-arounded the issue in the mean time by writing a small script that would call ``write_empty_vcf`` on the offending empty files.
